### PR TITLE
Sync host.v1 and host.v2. Add connectTimeout to host.v1. Fixes #1616

### LINKED ISSUE #1616: host.v1 and host.v2 are out of sync
host.v1 has listenOptions.connectTimeoutSeconds, host.v2 has listenOptions.connectTimeout.

Should allow both, preferring connectTimeout if available.

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

**Comment by ekoby** [comment]
is it just me or having `connectTimeout` on hosting configuration feels wrong?
What is it supposed to be -- listen/bind timeout?

**Comment by plorenz** [comment]
> is it just me or having `connectTimeout` on hosting configuration feels wrong? What is it supposed to be -- listen/bind timeout?

It's the connect timeout when making the outbound dial.

File: controller/persistence/migration_initialize.go [hunk 1]
```diff
@@ -338,7 +338,11 @@ var hostV1SchemaSansDefs = map[string]interface{}{
 				"properties": map[string]interface{}{
 					"connectTimeoutSeconds": map[string]interface{}{
 						"$ref":        "#/definitions/timeoutSeconds",
-						"description": "defaults to 5",
+						"description": "Timeout when making outbound connections. Defaults to 5. If both connectTimoutSeconds and connectTimeout are specified, connectTimeout will be used.",
```
Comment (scareything) [inline controller/persistence/migration_initialize.go:341]: does the validator that we use recognize the [deprecated annotation](https://json-schema.org/understanding-json-schema/reference/generic.html)?

**Review by scareything — APPROVED** [review]

File: controller/persistence/migration_initialize.go (hunk 1 above)
Comment (plorenz) [reply controller/persistence/migration_initialize.go:341]: Good idea, I'll give it a try :+1:

Comment (plorenz) [reply controller/persistence/migration_initialize.go:341]: done

Comment (scareything) [reply controller/persistence/migration_initialize.go:341]: are you going to update the rendered json for host.v1?

Comment (plorenz) [reply controller/persistence/migration_initialize.go:341]: I did, it's got the deprecated marker in it now

Comment (plorenz) [reply controller/persistence/migration_initialize.go:341]: oh, I misread that as host.v2. I'll add the updated host.v1 as well.

Comment (plorenz) [reply controller/persistence/migration_initialize.go:341]: updated host.v1

**Pull request merged**